addons/Request/data/settings.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" ?>
```

Comment:
Remove this file from the commit. It will (and should) be automatically generated.